src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -84,6 +84,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
       }
     } ~
     path("admin" / "groups" / Segment / "members") { (groupName) => //add members to group
+      put {
+        entity(as[RawlsGroupMemberList]) { memberList =>
+          requestContext => perRequest(requestContext,
+            UserService.props(userServiceConstructor, userInfo),
+            UserService.OverwriteGroupMembers(RawlsGroupRef(RawlsGroupName(groupName)), memberList))
```

Comment:
Not sure about this. From my understanding of HTTP spec, PUT should cover both create and overwrite. In which case I'm not sure why we need both it and POST.
http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html